release_manifests/24.10-0.7.0.0-0.yaml

```diff
@@ -0,0 +1,45 @@
+precompiled:
+- 24.10-0.7.0.0-0-5.15.0-25-generic-ubuntu22.04-amd64
+- 24.10-0.7.0.0-0-5.15.0-25-generic-ubuntu22.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-31-generic-ubuntu24.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-31-generic-ubuntu24.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1016-oracle-ubuntu22.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1016-oracle-ubuntu22.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1016-oracle-ubuntu24.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1016-oracle-ubuntu24.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1018-azure-ubuntu22.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1018-azure-ubuntu22.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1018-azure-ubuntu24.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1018-azure-ubuntu24.04-arm6
+- 24.10-0.7.0.0-0-6.8.0-1019-aws-ubuntu22.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1019-aws-ubuntu22.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1019-aws-ubuntu24.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1019-aws-ubuntu24.04-arm64
+dynamically_compiled:
+- 24.10-0.7.0.0-0-rhcos4.11-amd64
+- 24.10-0.7.0.0-0-rhcos4.12-amd64
+- 24.10-0.7.0.0-0-rhcos4.13-amd64
+- 24.10-0.7.0.0-0-rhcos4.13-arm64
+- 24.10-0.7.0.0-0-rhcos4.14-amd64
+- 24.10-0.7.0.0-0-rhcos4.14-arm64
+- 24.10-0.7.0.0-0-rhcos4.15-amd64
+- 24.10-0.7.0.0-0-rhcos4.15-arm64
+- 24.10-0.7.0.0-0-rhcos4.16-amd64
+- 24.10-0.7.0.0-0-rhcos4.16-arm64
+- 24.10-0.7.0.0-0-rhel8.6-amd64
+- 24.10-0.7.0.0-0-rhel8.6-arm64
+- 24.10-0.7.0.0-0-rhel8.8-amd64
+- 24.10-0.7.0.0-0-rhel8.8-arm64
+- 24.10-0.7.0.0-0-rhel8.9-amd64
+- 24.10-0.7.0.0-0-rhel8.9-arm64
+- 24.10-0.7.0.0-0-rhel8.10-amd64
+- 24.10-0.7.0.0-0-rhel8.10-arm64
+- 24.10-0.7.0.0-0-rhel9.0-amd64
+- 24.10-0.7.0.0-0-rhel9.2-amd64
+- 24.10-0.7.0.0-0-rhel9.3-amd64
+- 24.10-0.7.0.0-0-rhel9.4-amd64
+- 24.10-0.7.0.0-0-ubuntu20.04-amd64
+- 24.10-0.7.0.0-0-ubuntu22.04-amd64
+- 24.10-0.7.0.0-0-ubuntu22.04-arm64
+- 24.10-0.7.0.0-0-ubuntu24.04-amd64
+- 24.10-0.7.0.0-0-ubuntu24.04-arm64
```

Comment:
why not to have something like `ubuntu24.04: [amd64, arm64]`?

Reply:
for the dynamically compiled tags, this would indeed be clear and brief (since still quite simple).
however, for the precompiled tags (which one should expect to be in a consistent format across the file too), it would complicate the file's data structure, without providing any major advantages. here are the disadvantages, imho:
- less easy to read for humans.
- harder to implement in job that parses it (introducing slightly higher chance for bugs).
- cannot lookup a specific tag quickly (such as with `ctrl+f`).
- if one wants to perform a smoke detection test, such as "are there the expected amount of tags in the manifest?" counting them becomes harder too.

Reply:
this format isn't human-readable. it's really to hard to read it

Reply:
the alternative would require the human to assemble/concatenate each tag's string... much less human-friendly, imho...